Description: on Qt >= 4.5, debug info are disabled for QtWebkit.
             enable debug information in JavaScriptCore
Author: Fathi Boudra <fabo@debian.org>

---
 Source/JavaScriptCore/JavaScriptCore.pro |    2 --
 1 file changed, 2 deletions(-)

--- a/Tools/qmake/mkspecs/features/production_build.prf
+++ b/Tools/qmake/mkspecs/features/production_build.prf
@@ -9,8 +9,6 @@
 # symbols for the static libraries that implement API.
 linux-*|win32-g++*:if(CONFIG(debug, debug|release)|contains(QT_CONFIG,separate_debug_info)):contains(TEMPLATE, lib):staticlib {
     !equals(TARGET, WebKit2):!equals(TARGET, WebKit1) {
-        CONFIG += no_debug_info
-        CONFIG -= separate_debug_info
         QMAKE_CFLAGS += $$QMAKE_CFLAGS_RELEASE
         QMAKE_CXXFLAGS += $$QMAKE_CXXFLAGS_RELEASE
     }
